blocks/library/list/index.js

```diff
@@ -17,6 +17,47 @@ import { registerBlockType, createBlock } from '../../api';
 import Editable from '../../editable';
 import BlockControls from '../../block-controls';
 
+/**
+ * Converts an array of list items to an array of content.
+ *
+ * @param  {Array} items Array of React list items.
+ * @return {Array}       Array of Editable value arrays.
+ */
+function listItemsToArray( items ) {
+	return items.reduce( ( acc, item ) => {
+		// Ignore spacing between list items.
+		if ( typeof item === 'string' ) {
+			return acc;
+		}
+
+		const listContent = castArray( get( item, 'props.children', [] ) );
+
+		listContent.forEach( ( content, index ) => {
+			if ( content.type === 'ol' || content.type === 'ul' ) {
+				acc.push( ...listItemsToArray( castArray( get( content, 'props.children', [] ) ) ) );
+				acc.push( [] );
```

Comment:
Is `acc.push( [] );` required to have line breaks? maybe we specify in a comment the reason it is needed.